susemanager-utils/susemanager-sls/salt/packages/profileupdate.sls

```diff
@@ -10,12 +11,18 @@ packages:
           errors: report
 {%- endif %}
       }
+{% endif %}
 {% if grains['os_family'] == 'Suse' %}
 products:
   module.run:
     - name: pkg.list_products
 {% elif grains['os_family'] == 'RedHat' %}
 {% include 'packages/redhatproductinfo.sls' %}
+{% elif grains['os_family'] == 'Debian' %}
+ubunturelease:
```

Comment:
not sure if we want to be more neutral here and call it "debianrelease" as we do it for all in the family "Debian"

Reply:
Changed to `debianrelease`.